engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -1000,6 +1001,8 @@ namespace dmGameSystem
             write_ptr->m_MinorOrder = 0;
             write_ptr->m_MajorOrder = dmRender::RENDER_ORDER_WORLD;
             ++write_ptr;
+
+            DM_PROPERTY_ADD_U32(rmtp_Sprite, 1);
```

Comment:
Why do we add sprites one by one instead of adding `sprite_count` once?

Reply:
Not sure really. It seems like we do the same elsewhere.

Reply:
In some cases we exit function earlier because component disabled of similiar reason, but in this case it looks like we just operate each component in forloop, so it seems like we can use size instead (I'll fix it)